github/CommitComment.py

```diff
@@ -189,6 +190,21 @@ def create_reaction(self, reaction_type):
         )
         return github.Reaction.Reaction(self._requester, headers, data, completed=True)
 
+    def delete_reaction(self, reaction_id):
+        """
+        :calls: `DELETE /repos/:owner/:repo/comments/:comment_id/reactions/:reaction_id
+                <https://developer.github.com/v3/reactions/#delete-a-commit-comment-reaction>`_
+        :param reaction_id: integer
+        :rtype: None
+        """
+        assert isinstance(reaction_id, int), reaction_id
+        headers, data = self._requester.requestJsonAndCheck(
+            "DELETE",
+            self.url + "/reactions/" + str(reaction_id),
+            headers={"Accept": Consts.mediaTypeReactionsPreview},
+        )
+        return None
```

Comment:
Most delete actions check if the status returned is 204. Can you do that here?

Reply:
I can do this by calling `requestJson` instead. I actually followed the `delete` method of this class defined in previous lines. I guess the code is a bit old so that the method doesn't check the returned status. Do you think we should take them as tech debt and update these methods as well (in a separate PR)?